info.plist

```diff
@@ -105,11 +98,36 @@ require "net/http"
 require "uri"
 require "json"
 
-uri = URI.parse("https://pwpush.com/p.json")
+uri = URI.parse(ENV['site'] + "/p.json")
+
+retrieval_step = "false"
+if ENV['retrieval_step'] == "1" then
```

Comment:
And let's do the same for the others.  Here are the environment variables:
https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#password-push-expiration-settings